scipy/stats/_stats_py.py

```diff
@@ -922,11 +922,12 @@ def _moment_result_object(*args):
 # empty, there is no distinction between the `moment` function being called
 # with parameter `moments=1` and `moments=[1]`; the latter *should* produce
```

Comment:
Sorry missed this last time around. Does this comment also need updating? It looks like there might have originally been a moments->moment typo?

Reply:
Fixed, thanks.